### DISCUSSION TITLE
Seperate Read Article button is frustrating

### DISCUSSION THREAD

**Discussion by s-steephan** [dailydotdev/daily#653] [Feature Requests]
In the recent update we have to click on read article button to navigate to the post. It is actually frustrating. Old method was easy. Just click on post item opens article in new tab. You can bring a preference settings for it. So, old users will not feel difficulty to follow.

**Comment by idoshamun** [comment]
Hi @s-steephan ,

We're currently testing this new behavior. It's not completely rolled out.
Once and if we apply it to everyone, we might introduce a setting to change it back. Still in discussion, though.
We will also consider improving the experience of the read article button to make it easier in any way

**Comment by standingdreams** [comment]
@idoshamun How do I opt out of the behavior? Is there a feature flag that will allow us to opt out?

**Reply by idoshamun** [reply]
@standingdreams I can do that for you, but I can't promise how long it will last. Once we decide how to proceed, this might change.
Please give me your daily.dev username. You can also reach out to support at daily.dev if you want.

**Reply by standingdreams** [reply]
It's standingdreams

**Reply by idoshamun** [reply]
I excluded you from the experiment :)
